File: Uyanda.CoffeeShop.MicroService/Uyanda.CoffeeShop.Application/Features/BeverageManagement/Models/BeverageModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Uyanda.CoffeeShop.Application.Features.BeverageManagement.Models;

namespace Uyanda.CoffeeShop.Application.Features.BeverageManagement.Models
{
    public class BeverageModel
    {
        public int Id { get; set; }

        public int BeverageNameId { get; set; }

        public BeverageNameModel BeverageName { get; set; }

        public int BeverageTypeId { get; set; }

        public BeverageTypeModel BeverageType { get; set; }

       

        public int BeverageSizeId { get; set; }

        public BeverageSizeModel BeverageSize { get; set; }

        public decimal Price { get; set; }
    }
}

File: Uyanda.GUI.React.MicroService/react-app/src/models/beverage.model.ts
export interface Beverage {
  id: number;
  beverageNameId: number;
  beverageName: BeverageName;
  beverageTypeId: number;
  beverageType: BeverageType;
  beverageSizeId: number;
  beverageSize: BeverageSize;
  priceId: number;
  price: Price;
  count: number;
}

export interface BeverageName {
  id: number;
  name: string;
}

export interface BeverageType {
  id: number;
  name: string;
}

export interface BeverageSize {
  id: number;
  name: string;
}

export interface Price {
  id: number;
  currency: string;
  price: number;
}

export interface beverageProp {
  beverages: Beverage[];
}
